File: src/app/api/models/operation-lists-wrapper.ts
/* tslint:disable */
/* eslint-disable */
import { ExpenseDto } from '../models/expense-dto';
import { RevenueDto } from '../models/revenue-dto';
export interface OperationListsWrapper {
  listExpense?: Array<ExpenseDto>;
  listRevenue?: Array<RevenueDto>;
}

File: src/main/java/osiride/vitt_be/utils/OperationListsWrapper.java
package osiride.vitt_be.utils;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import osiride.vitt_be.dto.ExpenseDTO;
import osiride.vitt_be.dto.RevenueDTO;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class OperationListsWrapper {
	
	List<ExpenseDTO> listExpense;
	List<RevenueDTO> listRevenue;
}
